README.md

```diff
@@ -5,6 +5,18 @@ A small project that uses [mapit](http://mapit.mysociety.org) to query ward
 boundaries and show users whether their local government ward will change in
 2015.
 
+Quick install: Vagrant
+----------------------
+
+This project has a Vagrantfile, but before you can run it, you need to make sure the following things have been set up:
+
+1. Copy `conf/general.yml-example` to `conf/general.yml`, set `DJANGO_SECRET_KEY` to some random string of characters, and set `2015WARDS_DB_USER`, `2015WARDS_DB_NAME`, and `2015WARDS_DB_PASS` all to `'mapit'`.
```

Comment:
We could do away with this step by just putting those values into the general.yml-example and then copying it into the right place during the provisioning - it's not like we need to worry about security on someone's dev vagrant box..